lisa/tools/mount.py

```diff
@@ -11,7 +11,9 @@
 
 
 class Mount(Tool):
-    __UMOUNT_ERROR_PATTERN = re.compile(r".*mountpoint not found", re.MULTILINE)
+    __UMOUNT_ERROR_PATTERN = re.compile(
+        r"(.*mountpoint not found|.*no mount point specified)", re.MULTILINE
```

Comment:
The .* should be out of the capture group, it makes the regular expression more simple.

Reply:
fix